package.json

```diff
@@ -22,7 +22,7 @@
     "url": "http://github.com/home-assistant/homebridge-homeassistant/issues"
   },
   "engines": {
-    "node": ">=0.12.0",
+    "node": ">=6.0.0",
```

Comment:
In a later commit, I reduced this to ">=4.3.2", which is the version that introduced the class keyword. It's also the minimum version that's required by the homebridge package, so it doesn't actually represent anything different in what users will need to have set up.